### ISSUE TITLE
HTTPS

### ISSUE THREAD

**Issue by michaelerobertsjr** [flynn/flynn#4117]
I am having difficulty getting HTTPS working.  

We were able to get the app up and working at a url similar to: http://myapp.app.domain.com

## DNS settings

A records are setup for domain.com for all three flynn node instances
CNAME records exist for app.domain.com
CNAME records exist for *.app

## Steps to reproduce issue

Use ACME app to generate certificates:

```
acme cert -dns myapp.app.domain.com
```

On local machine use:

```
flynn route update http/some-long-hash --tls-cert myapp.app.domain.crt --tls-key myapp.app.domain.key
```

## Expected result

Should be able to now use https://myapp.app.domain.com

Instead we are getting:

This site cant be reached

DNS_PROBE_FINISHED_NXDOMAIN

**Comment by titanous** [comment]
That error indicates that your DNS is misconfigured. If you can provide the domain names in question we can check if there's anything obvious, but beyond that I'd suggest looking very closely at your DNS configuration to ensure everything is correctly set up.

**Comment by michaelerobertsjr** [comment]
Thanks, but the odd part is that I didn't change any dns settings.  This app was working fine with http and I had a similar issue with another app.  Both were working fine before adding the https routes to the apps.  

Doesn't that eliminate dns records setup on our dns provider as the root cause of this issue?

**Comment by titanous** [comment]
The error points at DNS. Without you posting the actual domain in question I'm unable to help further.

**Comment by michaelerobertsjr** [comment]
@titanous Thanks for offering to help, but this is the same issue I messaged you in IRC before about another app.  You said that was DNS as well.  I can't understand why our app works with http, and suddenly it is a DNS issue on our side when we follow the Flynn guide to add routes... Anyway, this is just not an efficient use of our time.

Unfortunately we have to move on from Flynn.  It is just to difficult for us to resolve issues. We are developers that want to write code, not solve DevOps challenges, so Flynn sounds amazing, but I've spent several business days trying to figure things out.

I put all our infrastructure up in about 2 hours on Heroku (including the entire Continuous Delivery pipeline).  THAT is what it should have been like with Flynn!

This is my second attempt at using Flynn on a project.  I looked at Flynn a year ago and was really impressed by the dashboard that was provided to initially help bootstrap instances.  Disappointed that is gone now as well.

I would be excited to take another look when Flynn is more mature, there is a community that can provide support, and more documented implementations of https. Right now it is really frustrating that a simple implementation is too difficult.  This project needs way more community support and a more reasonable support package offering.

So disappointing...
